junction/templates/proposals/detail/base.html

```diff
@@ -180,6 +180,13 @@ <h4 class='heading'><b>Speaker Links:</b></h4>
           </div>
           {% endif %}
 
+          {% if proposal.is_first_time_speaker %}
+          <div class="proposal-writeup--section">
+            <h4 class='heading'><b>First Time Speaker:</b></h4>
+            <p> Yes</p>
```

Comment:
Do we use any library like font-awesome? Instead of yes we can show a :heavy_check_mark: